fontbro/font.py

```diff
@@ -271,17 +271,17 @@ def get_characters(self, ignore_blank=False):
         glyfs = font.get("glyf")
         for code, char_name in cmap.items():
             code_hex = f"{code:04X}"
-            char = chr(code)
+            if code in range(0x110000):
```

Comment:
a `code_range` variable could be created outside the loop to avoid range object creation at every loop iteration.

Reply:
I changed it to `if 0 <= code < 0x110000:` which should be cleaner.